packages/env/README.md

```diff
@@ -8,25 +8,60 @@ Ensure that Docker is running, then:
 
 ```sh
 $ cd /path/to/a/wordpress/plugin
-$ npx wp-env start
+$ npm -g i @wordpress/env
+$ wp-env start
 ```
 
 The local environment will be available at http://localhost:8888.
 
-## Instructions
-
-### Installation
+## Prerequisites
 
 `wp-env` requires Docker to be installed. There are instructions available for installing Docker on [Windows 10 Pro](https://docs.docker.com/docker-for-windows/install/), [all other versions of Windows](https://docs.docker.com/toolbox/toolbox_install_windows/), [macOS](https://docs.docker.com/docker-for-mac/install/), and [Linux](https://docs.docker.com/v17.12/install/linux/docker-ce/ubuntu/#install-using-the-convenience-script).
 
-After confirming that Docker is installed, you can install `wp-env` globally like so:
+`wp-env` is also distributed through NPM (Node Package Manager), so Node.js and NPM are required. The latest LTS version of Node.js is used to develop `wp-env` and is recommended.
+
+## Installation
+
+### Installation as a global package
+
+After confirming that the prerequisites are installed, you can install `wp-env` globally like so:
 
 ```sh
 $ npm -g i @wordpress/env
 ```
 
 You're now ready to use `wp-env`!
 
+### Installation as a local package
+
+If your project already has a package.json, it's also possible to use `wp-env` as a local package. First install `wp-env` locally as a dev dependency:
+
+```sh
+$ npm i @wordpress/env --save-dev
+```
+
+Then modify your package.json and add an extra command to npm `scripts` (https://docs.npmjs.com/misc/scripts):
+
+```json
+"scripts": {
+	"wp-env": "packages/env/bin/wp-env"
```

Comment:
seems like we don't require this `npm run wp-env start` without this works
.

Reply:
Good point about the path. I think this may be needed for more advanced usage to be able to add extra arguments to a command e.g.
```
npm run wp-env run composer run-script lint
```

but I haven't tested that. Possibly the path would have to change to the `node_modules` folder, or maybe it does already work without further changes.

Reply:
```
 "scripts": {
    "env:start": "wp-env start",
    "env:stop": "wp-env stop"
  }
```
this works fine without specifying path to command

Reply:
I've removed that part now @ajitbohra, thanks for the feedback 👍

Reply:
>I've removed that part now @ajitbohra, thanks for the feedback 👍

I think he meant to change `"wp-env": "packages/env/bin/wp-env"` to `"wp-env": "wp-env"`.

We still need that there otherwise `npm run` won't resolve the command. In Gutenberg, we use a local path because we want to use the checked-out version of the package for development.